File: Projects/4-Guessing-Game.py
from random import randint

secret = randint(1, 10)

while True:
    guess = int(input("Pick a number from 1 to 10: "))
    
    if guess < secret:
        print("Too low, try again!")
    elif guess > secret: 
        print("Too high, try again!")
    else:
        print("You guessed it! You won!")

        play_again = input("Do you want to play again? (y/n) ")
        if play_again == "y":
            secret = randint(1, 10)
        else:
            print("Thank you for playing!")
            break

File: Projects/4-Guessing-Game.js
const prompt = require('prompt-sync')()
const random = require('lodash/random')

let secretNumber = random(1, 10)

while (true) {
    let guess = Number(prompt("Pick a number between 1 and 10: "))

    if (guess < secretNumber) {
        console.log("Too low, try again!")
    } else if (guess > secretNumber) {
        console.log("Too high, try again!")
    } else {                                        
        console.log("You guessed it! You won!")

        let playAgain = prompt("Do you want to play again? (y/n) ")
        if (playAgain === 'y') {
            secretNumber = random(1, 10)
        } else {
            console.log("Thank you for playing!")
            break
        }
    }
}
